ci-operator/step-registry/openshift-qe/move-pods-infra/openshift-qe-move-pods-infra-commands.sh

```diff
@@ -63,9 +63,14 @@ function check_monitoring_statefulset_status()
   for statefulset in $statefulset_list; do
     echo "statefulset in openshift-monitoring is $statefulset"	  
     retries=0
-    ready_replicas=$(oc get statefulsets $statefulset -n openshift-monitoring -ojsonpath='{.status.availableReplicas}')
     wanted_replicas=$(oc get statefulsets $statefulset -n openshift-monitoring -ojsonpath='{.spec.replicas}')
+    echo wanted_replicas is $wanted_replicas
+    sleep 30
```

Comment:
Is this just a band-aid? Could it still fail IF the field isn't there after 30s ? Wondering if there's some kind of definitive, null-safe check we can do if the field is there, and then check the field.

Reply:
@afcollins Thank you for reviewing my code, Yes, if .status.availableReplicas return nothing will still failure after sleep 30s because of we set `set -o errexit`, sometime it will take second to update the status. most of time, the value will be updated very fast. except adding sleep time,  I add checking field steps in my new code per your comments. please review again.

Reply:
Ok, sure. If 30s is much longer than we ever expect the field to be available, then that sounds fine. My interest is in removing all possible scripting errors since by the time this step runs, all worker nodes have already been provisioned.

With the change below at L70, you are checking 'readyReplicas' and then 'availableReplicas'. Just want to confirm that I am seeing two different values there.

Reply:
Thank you for pointing the error, that's wrong, I have correct to 'availableReplicas'.